server/controllers/user.js

```diff
@@ -19,13 +24,68 @@ class UserController {
     }
   };
 
+  getUserViaEmail = async (req, res, next) => {
+    try {
+      const userEmail = req.params.email;
+      const user = await this.userService.getUserViaEmail(req.params.email);
+      if (user == null) {
+        next(ApiError.notFound(`User with email ${userEmail} not found`));
+        return;
```

Comment:
What's the purpose of this return statement?

Reply:
If it's just to exit before the 200 below, then I think maybe an if-else could do the same job (and perhaps be less susceptible to bugs)

Reply:
Hmm good point! I'm not sure actually, but I was just following the getUser method. It does work with and without it. 
We could remove it. @jcsnorlax97 What do you think?

Reply:
Yeah. My bad. I think we can remove this `return`.